src/components/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import { forwardRef, Dispatch } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
```

Comment:
Don't pass more information, than you need to the component
```suggestion
  hasTodos: boolean;
```